components/contribute-cards/ContributeTiersPanel.js

```diff
@@ -0,0 +1,155 @@
+import ContributeCustom from './ContributeCustom';
+import LoadingPlaceholder from '../LoadingPlaceholder';
+import { CONTRIBUTE_CARD_WIDTH } from './Contribute';
+import colors from '../../lib/constants/colors';
+
+const Handle = styled.div`
+  background-color: ${({ bgColor }) => bgColor || colors.white};
+  box-shadow: 1px 1px 1px ${colors.darkgray};
+  opacity: ${({ hide }) => (hide ? 0 : 1)};
+  transition: all 0.3s ease-out;
+  justify-content: center;
+  color: ${colors.white};
+  align-items: center;
+  margin-bottom: 5px;
+  border-radius: 50%;
+  outline: none;
+  display: flex;
+  height: 30px;
+  width: 30px;
+  > * {
+    color: ${({ color }) => color || colors.darkgray};
+  }
+`;
+
+const ContributeTiersPanel = ({
+  isAdmin,
+  collective,
+  sortedTiers,
+  hasNoContributor,
+  contributorsStats,
+  handleSettingsUpdate,
+  CONTRIBUTE_CARD_PADDING_X,
+  createContributionTierRoute,
+  financialContributorsWithoutTier,
+}) => {
+  const CONTRIBUTE_CARD_HEIGHT = 341;
+
+  const StyledDragDropPlaceHolder = () => (
+    <LoadingPlaceholder width={CONTRIBUTE_CARD_WIDTH} heigth={CONTRIBUTE_CARD_HEIGHT} />
+  );
+  const dynamicOptions = { loading: StyledDragDropPlaceHolder, ssr: true };
+  const StyledDragDrop = dynamic(
+    () => import(/* webpackChunkName: 'StyledDragDrop' */ '../StyledDragDrop'),
+    dynamicOptions,
+  );
+
+  const getTiersOrder = memoizeOne(() => {
+    return get(collective, 'settings.collectivePage.tiersOrder', []).filter(i => i !== 'custom');
+  });
+
+  const handleShuffle = newTiersOrder => {
+    const settings = get(collective, 'settings', {});
+
+    set(settings, 'collectivePage.tiersOrder', concat(['custom'], newTiersOrder));
+
+    handleSettingsUpdate(settings);
+  };
+
+  if (!isAdmin) {
+    return (
+      <Fragment>
+        <Box px={CONTRIBUTE_CARD_PADDING_X}>
+          <ContributeCustom
+            collective={collective}
+            contributors={financialContributorsWithoutTier}
+            stats={contributorsStats}
+            hideContributors={hasNoContributor}
+          />
+        </Box>
+        {sortedTiers.map(tier => (
+          <Box key={tier.id} px={CONTRIBUTE_CARD_PADDING_X}>
+            <ContributeTier collective={collective} tier={tier} hideContributors={hasNoContributor} />
+          </Box>
+        ))}
+      </Fragment>
+    );
+  } else
+    return (
+      <Fragment>
+        <Box px={CONTRIBUTE_CARD_PADDING_X}>
+          <CreateNew data-cy="create-contribute-tier" route={createContributionTierRoute}>
+            <FormattedMessage id="Contribute.CreateTier" defaultMessage="Create Contribution Tier" />
+          </CreateNew>
+        </Box>
+        <Box px={CONTRIBUTE_CARD_PADDING_X}>
+          <ContributeCustom
+            collective={collective}
+            contributors={financialContributorsWithoutTier}
+            stats={contributorsStats}
+            hideContributors={hasNoContributor}
+          />
+        </Box>
+        <StyledDragDrop
```

Comment:
`StyledDragDrop` should only be used when the users - aka when they can actually move the cards. We don't want this (heavy) component to be imported for all visitors.